src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+import duke.DukeException;
+
+public class Todo extends Task {
+
+    public Todo(String description) throws DukeException {
+        super(description);
+    }
+
+    protected String getTypeNameWithQuantifier() {
+        return "a todo";
+    }
+
+    public String toExportFormat() {
+        return "T | " + (this.isDone ? "1" : "0") + " | " + this.description;
```

Comment:
toExportFormat() seems to work alright, perhaps you have your own reasoning to make toExportFormat abstract, on the other hand, you can also consider shifting "| " + (this.isDone ? "1" : "0") + " | " + this.description to Task's toExportFormat instead and use it in its children classes to reduce repetition of code